src/components/Header/NavBar/NavBar.js

```diff
@@ -6,24 +6,48 @@ import ConnectedAuthorization from './Authorization/Authorization';
 export default class NavBar extends Component {
   render() {
     return (
-      <nav className="NavBar">
-        <ul>
-          <li>
-            <NavLink to="/" exact>Home</NavLink>
-          </li>
-          <li>
-            <NavLink to="/cards">Cards</NavLink>
-          </li>
-          <li>
-            <NavLink to="/products">Products</NavLink>
-          </li>
-          <li>
-            <ConnectedAuthorization
-              withAuth={logOut => <NavLink to="/login" onClick={logOut}>Log out</NavLink>}
-              withoutAuth={<NavLink to="/login">Login</NavLink>}
-            />
-          </li>
-        </ul>
+      <nav>
+        <div className="NavBar">
+          <ul className="NavBar__ul">
+            <li className="NavBar__ul__li">
+              <NavLink className="NavBar__ul__li__a" to="/" exact>Home</NavLink>
+            </li>
+            <li className="NavBar__ul__li">
+              <NavLink className="NavBar__ul__li__a" to="/cards">Cards</NavLink>
+            </li>
+            <li className="NavBar__ul__li">
+              <NavLink className="NavBar__ul__li__a" to="/products">Products</NavLink>
+            </li>
+            <li className="NavBar__ul__li">
+              <ConnectedAuthorization
+                withAuth={logOut => <NavLink className="NavBar__ul__li__a" to="/login" onClick={logOut}>Log out</NavLink>}
+                withoutAuth={<NavLink className="NavBar__ul__li__a" to="/login">Login</NavLink>}
+              />
+            </li>
+          </ul>
+        </div>
+        <div className="NavBarNarrow">
```

Comment:
`...Narrow` sounds like a modificator to me, probably `NavBar--narrow` would be better